test/modelValidatorTests.js

```diff
@@ -123,4 +123,18 @@ describe('Model Validation', function () {
       });
     });
   });
+
+  describe('for parameters in x-ms-parameterized-host', function () {
```

Comment:
should we also have a test on semantic validation, to make sure we're not breaking it, particularly since we have the caveat that we shouldn't be applying the changes to semantic validation?

Reply:
added a semantic validation test as well.